sinon/lib/stub.py

```diff
@@ -17,118 +17,178 @@ class SinonStub(SinonSpy):
     (4) a module
 
     All function/module will be replaced into customized or empty function/class
-    Able to be assigned special condition (on which call, with what args...etc)
+    Able to be assigned special conditions (on which call, with what args, etc.)
     """
 
     def __init__(self, obj=None, prop=None, func=None):
         super(SinonStub, self).__init__(obj, prop)
-        self.stubfunc = func if func else Wrapper.empty_function
-        super(SinonStub, self).wrap2stub(self.stubfunc)
-        # Todo: target is a dirty hack
```

Comment:
please keep this line there, this [dirty hack](https://github.com/note35/sinon/commit/984183d46ad8cfab6c5bb472f0a5f15030e1d2c2) is still exist after your modification.

Reply:
Why do you consider this to be a dirty hack? Because a reference to the target is not a condition?